strawberry_django/filters.py

```diff
@@ -49,7 +52,7 @@
 
 @strawberry.input
 class DjangoModelFilterInput:
-    pk: strawberry.ID
+    pk: strawberry.ID  # TODO: How to override pk here with settings["DEFAULT_PK_FIELD_NAME"] ?
```

Comment:
In theory we could do:

```python

class DjangoModelFilterInput:
    pass

DjangoModelFilterInput.__annotations__[settings["DEFAULT_PK_FIELD_NAME"]] = strawberry.ID
DjangoModelFilterInput = strawberry.input(DjangoModelFilterInput)
```

But that would mean forcing django to be imported earlier (which can lead to some issues). I'll try to think if there's a more elegant solution.

What do you think @patrick91 ?

Reply:
I can wrap it in a function to prevent the import from Module scope